addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
@hhff I think the point of the `infinityModel` param is to use it when overriding without accessing anything private.